src/handlers/message.ts

```
@@ -40,11 +42,50 @@ export class MessageHandler {
+                if (existingMessage) {
+                  await this.handleMessageEdit(originalMessageId, protocolMsg.editedMessage, existingMessage);
+                } else {
+                  logger.warn('Message edit received for unknown message', { originalMessageId });
+                  debugLogger.warn('Message edit for unknown message', { originalMessageId, waMessage });
+                }
+              }
+            }
+            break;
+
+          case proto.Message.ProtocolMessage.Type.REVOKE:
+            const originalMessageId = protocolMsg.key?.id;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)